dss/operations/secrets.py

```diff
@@ -0,0 +1,310 @@
+    },
+)
+def list_secrets(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Print a list of names of every secret variable in the secrets manager
+    for the given store and stage.
+    """
+    store_prefix = get_secretsmanager_prefix(args)
+
+    paginator = secretsmanager.get_paginator("list_secrets")
+
+    secret_names = []
+    for response in paginator.paginate():
+        for secret in response["SecretList"]:
+
+            # Get resource IDs
+            secret_name = secret["Name"]
+
+            # Only save secrets for this store and stage
+            if secret_name.startswith(store_prefix):
+                secret_names.append(secret_name)
+
+    secret_names.sort()
+
+    if hasattr(args, "json") and args.json is True:
+        print(json.dumps(secret_names))
+    else:
+        for secret_name in secret_names:
+            print(secret_name)
+
+
+@events.action(
+    "get",
+    arguments={
+        "--secret-names": dict(
+            required=True,
+            nargs="*",
+            help="names of secrets to retrieve (separate multiple values with spaces)",
+        ),
+        "--json": dict(
+            default=False,
+            action="store_true",
+            help="format the output as JSON if this flag is present",
+        ),
+    },
+)
+def get_secret(argv: typing.List[str], args: argparse.Namespace):
+    """
+    Get the value of the secret variable (or list of variables) specified by
+    the --secret-names flag; separate multiple secret names using a space.
+    """
+    # Note: this function should not print anything except the final JSON,
+    # in case the user pipes the JSON output of this script to something else
+
+    store_prefix = get_secretsmanager_prefix(args)
+
+    # Make sure something was specified for secret name(s)
+    if len(args.secret_names) == 0:
+        raise RuntimeError(
+            "Unable to set secret: no secret name was specified! Use the --secret-names flag."
+        )
+    else:
+        secret_names = args.secret_names
+
+    # Tack on the store prefix if it isn't there already
+    for i in range(len(secret_names)):
+        secret_name = secret_names[i]
+        if not secret_name.startswith(store_prefix):
+            secret_names[i] = store_prefix + secret_name
+
+    # Determine if we should format output as JSON
+    use_json = False
+    if hasattr(args, "json"):
+        if args.json:
+            use_json = True
+
+    for secret_name in secret_names:
+        # Attempt to obtain secret
+        try:
+            response = secretsmanager.get_secret_value(SecretId=secret_name)
+            secret_val = response["SecretString"]
+        except ClientError:
+            # A secret variable with that name does not exist
+            logger.warning(f"Resource not found: {secret_name}")
+        else:
+            # Get operation was successful, secret variable exists
+            if use_json:
+                print(json.dumps({secret_name: secret_val}))
```

Comment:
Consider using `, indent=4` again, and also, sometimes the `secret_val` is a dictionary, but this will print it as a string, so consider using something like:
```
try:
    secret_val = json.loads(secret_val)
except:
    pass

print(json.dumps({secret_name: secret_val}, indent=4))
```

For reference, the output of the command: `scripts/dss-ops.py secrets get --secret-name dcp/dss/dev/admin_user_emails dcp/dss/dev/application_secrets.json --json`

Reply:
Done, thanks!